heat/core/manipulations.py

```diff
@@ -1724,6 +1724,10 @@ def reshape(a: DNDarray, shape: Tuple[int, ...], new_split: Optional[int] = None
     if not isinstance(a, DNDarray):
         raise TypeError("'a' must be a DNDarray, currently {}".format(type(a)))
 
+    # check for nested sequence
+    if isinstance(shape[0], (tuple, list)):
+        shape = shape[0]
+
```

Comment:
This is so slow, please @Markus-Goetz @coquelin77 can you suggest a more efficient way?